src/modules/block.js

```diff
@@ -110,4 +110,4 @@ module.exports = bot => {
       unblock(thread.user_id);
     }
   });
-};
+};
```

Comment:
EOF newline

Reply:
Where lol (same for the `notes.js` file)

Reply:
You're missing one